header_test.go

```diff
@@ -2922,4 +2922,4 @@ func expectResponseHeaderAll(t *testing.T, h *ResponseHeader, key string, expect
 	if !reflect.DeepEqual(h.PeekAll(key), expectedValue) {
 		t.Fatalf("Unexpected value for key %q: %q. Expected %q", key, h.PeekAll(key), expectedValue)
 	}
-}
+}
```

Comment:
Why are you removing all the newlines at the end of files? They should be there.

Reply:
I use goland ide, the format is different from github lint, i will fix it soon